compute/etc/sql_exporter/compute_backpressure_throttling_seconds.libsonnet

```diff
@@ -1,6 +1,6 @@
 {
   metric_name: 'compute_backpressure_throttling_seconds',
-  type: 'gauge',
+  type: 'counter',
```

Comment:
@knizhnik am I right that `neon.backpressure_throttling_time()` is actually a counter internally? I.e. it never goes down